test/broadway_rabbitmq/producer_test.exs

```diff
@@ -151,6 +151,29 @@ defmodule BroadwayRabbitMQ.ProducerTest do
     assert options[:buffer_size] == 60
   end
 
+  test "producer :buffer_size and :buffer_keep can be " do
```

Comment:
"... can be overridden" ?

Reply:
Look at that, we haven't event met in person and you can already finish my sentences. ;)

Reply:
@msaraiva force-pushed the end of the sentence, there were no other changes.